packages/main/src/themes/ToggleButton.css

```diff
@@ -2,13 +2,18 @@
 	display: inline-block;
 }
 
+:host([design="Emphasized"]:not([pressed])) {
+	text-shadow: var(--_ui5_toggle_button_emphasized_text_shadow);
+}
+
 :host([pressed]),
 :host([design="Default"][pressed]),
 :host([design="Transparent"][pressed]),
 :host([design="Emphasized"][pressed]) {
 	background: var(--sapButton_Selected_Background);
 	border-color: var(--sapButton_Selected_BorderColor);
 	color: var(--sapButton_Selected_TextColor);
+	text-shadow: none;
```

Comment:
Can't we use the same CSS variable here var(--_ui5_toggle_button_emphasized_text_shadow);

Reply:
ACK